tests/pytest_framework/src/syslog_ng_config/statements/logpath/logpath.py

```diff
@@ -27,13 +27,17 @@
 class LogPath(object):
     def __init__(self):
         self.logpath_id = "logpath_%s" % RandomId(use_static_seed=False).get_unique_id()
-        self.full_logpath_node = {self.logpath_id: {"sources": [], "destinations": [], "flags": []}}
+        self.full_logpath_node = {self.logpath_id: {"sources": [], "filters": [], "destinations": [], "flags": []}}
         self.logpath_node = self.full_logpath_node[self.logpath_id]
 
     def add_source_groups(self, source_groups):
         target_node = self.logpath_node["sources"]
         self.update_logpath_node(target_node, source_groups)
 
+    def add_filter_groups(self, filter_groups):
```

Comment:
A pattern is evolving here too. These four functions do identical things except a string that specifies the target node. Please eliminate the repetition.

Reply:
+1, will be fixed.